cranelift/codegen/src/isa/x64/inst/emit_tests.rs

```diff
@@ -1529,6 +1529,19 @@ fn test_x64_emit() {
         "4F8D840AB3000000",
         "lea     179(%r10,%r9,1), %r8",
     ));
+    insns.push((
+        Inst::lea(Amode::rip_relative(BranchTarget::ResolvedOffset(0)), w_rdi),
+        "488D3D00000000",
+        "lea     (offset 0)(%rip), %rdi",
```

Comment:
Suggestion: could we make the `ShowWithRRU` pretty-print format here use assembler-parseable syntax, i.e., `lea 0(%rip), %rdi`? This makes it slightly more mechanical (hence easier to do by hand, and easier to possibly automate later) to verify that the golden bytes match the instruction.